src/hotspot/share/cds/aotArtifactFinder.cpp

```diff
@@ -243,6 +248,11 @@ void AOTArtifactFinder::add_cached_instance_class(InstanceKlass* ik) {
       add_cached_instance_class(intf);
     }
 
+    InstanceKlass* nest_host = ik->nest_host_or_null();
```

Comment:
Just curious, what is this nest host treatment for? I don't think the generated Holder classes have the NestHost attribute, despite them being listed in NestMembers of their top level classes.